python_modules/libraries/dagster-databricks/dagster_databricks/ops.py

```diff
@@ -135,3 +136,239 @@ def create_ui_url(databricks_client, op_config):
         return "https://{host}/?o={workspace_id}#joblist".format(
             host=host, workspace_id=workspace_id
         )
+
+
+def create_databricks_run_now_op(databricks_job_id: int, **kwargs) -> OpDefinition:
+    """
+    Creates an op that launches an existing databricks job.
+
+    As config, the op accepts a blob of the form described in Databricks' Job API:
+    https://docs.databricks.com/api-explorer/workspace/jobs/runnow. The only required field is
+    ``job_id``, which is the ID of the job to be executed. Additional fields can be used to specify
+    override parameters for the Databricks Job.
+
+    Arguments:
+        databricks_job_id (int): The ID of the Databricks Job to be executed.
```

Comment:
I know you mentioned it above, but I think it's worth mentioning the kwargs here and being specific about what kinds of parameters are accepted (all the parameters to the op decorator?)